keymaps/linux.cson

```diff
@@ -143,6 +143,10 @@
   'ctrl-k ctrl-8': 'editor:fold-at-indent-level-8'
   'ctrl-k ctrl-9': 'editor:fold-at-indent-level-9'
 
+'atom-workspace atom-pane':
+  'ctrl-alt-=': 'pane:enlarge'
+  'ctrl-alt--': 'pane:shrink'
```

Comment:
Should these be `pane:increase-size` and `pane:decrease-size`, same as in the OSX keybindings?